setup.py

```diff
@@ -66,7 +66,6 @@
         "ms2pip.cython_modules.ms2pip_pyx",
         sources=["ms2pip/cython_modules/ms2pip_pyx.pyx"] + glob("ms2pip/models/*/*.c"),
         extra_compile_args=[
-            "-fno-var-tracking-assignments",
```

Comment:
As we currently always do a build with debugging symbols, the use of `-fno-var-tracking-assignments` makes sense. I guessed you removed it because the compiler for the MacOS X builds doesn't support it, but the commit where you made this change suggests that there might be another reason.

Reply:
I indeed removed it just for macOS support. So would it be better to let this flag depend on the OS?

Reply:
Comparing GH Actions build steps with and without the flag don't really show a noticeable difference in build time:
- With no-VTA flag ([d819e98](https://github.com/compomics/ms2pip_c/commit/d819e9897da041f9a1dbcdca284bb9411705d7c4)): 7m 11s, 7m 4s, and 6m 56s
- Without no-VTA flag ([416d733](https://github.com/compomics/ms2pip_c/runs/1030621602)): 6m 35s, 7m 41s, and 7m 19s

Reply:
Let's drop it.